wordpress-mobile/WordPress-Android
# View all followers

This is an intermediate PR that only adds a new View all screen for the followers (too keep the PRs small).

To test:
1. Go to Stats
2. Find the Followers block and tap View more
3. Notice a new activity is opened and 10 items are immediately loaded
4. Tap on the Load more button at the bottom
5. Notice 10 more items are loaded
6. Keep tapping the button until all available items are loaded
7. After everything is loaded on the screen, the Load more button is gone

![image](https://user-images.githubusercontent.com/1522856/52418059-8c3e0000-2aed-11e9-9e2c-482ad09be5a1.png)
